static/js/color_class.js

```diff
@@ -0,0 +1,57 @@
+import _ from "lodash";
+
+import * as colorspace from "./colorspace";
+
+export function get_threshold() {
+    // sRGB color component for dark label text.
+    // 0x33 to match the color #333333 set by Bootstrap.
+    const label_color = 0x33;
+    const lightness = colorspace.luminance_to_lightness(colorspace.sRGB_to_linear(label_color));
+
+    // Compute midpoint lightness between that and white (100).
+    return (lightness + 100) / 2;
```

Comment:
Hrm, maybe we should just hard code this value rather than computing it at startup?  (And then we could have a unit test to verify it.)  I think that's a bit out of the scope of the extract commit, but seems like a good idea.

Reply:
I'm not sure it'd be worth the time to do that; this function looks really fast and only runs once on startup, so I don't really see the downside to the current structure.